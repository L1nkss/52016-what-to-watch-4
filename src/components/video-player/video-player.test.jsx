import VideoPlayer from "./video-player";

const mockOne = {
  poster: `img/bohemian-rhapsody.jpg`,
  isActive: false,
  src: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};
const mockTwo = {
  poster: `img/bohemian-rhapsody.jpg`,
  isActive: true,
  src: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

describe(`Testing VideoPlayer component`, () => {
  it(`Component should successfully rendered if isActive is false`, () => {
    const component = renderer.create(<VideoPlayer poster={mockOne.poster} isActive={mockOne.isActive} src={mockOne.src}/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  it(`Component should successfully rendered if isActive is true`, () => {
    const component = renderer.create(<VideoPlayer poster={mockTwo.poster} isActive={mockTwo.isActive} src={mockTwo.src}/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
