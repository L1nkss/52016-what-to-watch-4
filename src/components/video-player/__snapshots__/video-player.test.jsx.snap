// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing VideoPlayer component Component should successfully rendered if isActive is false 1`] = `
<video
  className="player__video"
  muted={true}
  poster="img/bohemian-rhapsody.jpg"
/>
`;

exports[`Testing VideoPlayer component Component should successfully rendered if isActive is true 1`] = `
<video
  className="player__video"
  muted={true}
  poster="img/bohemian-rhapsody.jpg"
/>
`;
